src/lib/theme.js

```diff
@@ -16,7 +16,7 @@ export const spacing = {
   xl: '40px',
   xxl: '52px',
   xxxl: '96px',
-  xxxxl: '240px',
+  xxxxl: '104px',
```

Comment:
Not sure about this change -- if the `240px` value is already being used somewhere else then we should keep it. Also, having the difference between `xxxl` and `xxxxl` be 8 pixels seems a bit silly. What if we either hard coded `104px` for the welcome screen margins, or just used `xxxl` and not tell anyone that we skimped on the last 8 pixels?

Reply:
Agree with @mulias -- also, as the number of `x`s increases, it gets harder to tell one value from another (e.g., `xxxxxxxxxxxxxl` vs `xxxxxxxxxxxxxxl`). I advocate for removing the `xxxxl` value.

Reply:
ok sounds good, the 240px was used in the last Welcome screen design so it is not used anywhere else, but I agree the `xxxxx`'s become messy fast